soc/nxp/lpc/lpc55xxx/Kconfig.defconfig

```diff
@@ -0,0 +1,94 @@
+# Copyright 2019,2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_LPC55XXX
+
+config NUM_IRQS
+	# must be >= the highest interrupt number used
+	default 60
+
+config INIT_PLL0
+	bool "Initialize PLL0"
+
+config INIT_PLL1
```

Comment:
I think we want to define new Kconfigs in the SOC `Kconfig` file, and only set defaults in `Kconfig.defconfig`

Reply:
Fixed